<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewMode</class>
 <widget class="QDialog" name="viewMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change View Mode</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>View Modes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="viewModeList">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="flow">
         <enum>QListView::LeftToRight</enum>
        </property>
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>Remove Mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Toolboxes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="_10">
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>&amp;Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="text">
            <string>&amp;Ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="_7">
         <item row="0" column="0">
          <widget class="QListWidget" name="toolboxList">
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::NoDragDrop</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="movement">
            <enum>QListView::Static</enum>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="_8">
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="upButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="removeWidgetButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="downButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save Configuration</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Widget</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Toolbars</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
